tests/Fixtures/e2e/Test_Frameworks/run_tests.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+#Run test suite with phpunit
+php ../../../../bin/infection
+if [ $? != 0 ]
```

Comment:
you can put `set -euf -o pipefail` at the beginning of the file instead of having to rely on those return values each time

If you do, it also becomes unnecessary to do the whole checks at each `diff` calls as well, you can just chain things together and if an error occurs you should know where exactly from the error

Just to be clear: what you did is perfectly correct and valid, I think the above however would simplify this script

Reply:
The same applies to `tests/e2e_tests`

Reply:
Great, i didn't know that was an option.